src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +38,16 @@ public class RobotContainer {
      */
     public RobotContainer() {
         SmartDashboard.putData("Choose Auto: ", autoChooser);
-        autoChooser.setDefaultOption("Do Nothing", new WaitCommand(1));
+        autoChooser.setDefaultOption("Wait 1 Second", "wait");
+        if (Robot.isReal()) {
+            // Use Real HW
+            drivetrain = new Drivetrain(new DrivetrainIO() {});
```

Comment:
This should probably reference the real DrivetrainIO then (`DrivetrainVictorSP`)

Reply:
100%, copy and paste error